### PR TITLE
Clarify first seen help text

### PR DESCRIPTION
**Pull request by wlach**
It was a bit ambiguous what this meant, hopefully this clarifies things somewhat.

### CODE REVIEW

**Comment by wlach** [comment]
@Iinh @chutten how does this sound to you?

File: src/data/schemas.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ const REQUIRED_METRIC_PARAMS_DOCS =
 const OPTIONAL_METRIC_PARAMS_DOCS =
   "https://mozilla.github.io/glean/book/user/metric-parameters.html#optional-metric-parameters";
 
+const getFirstSeenTemplate = (itemType) => {
+  return `The date when this ${itemType} was first added to the product's source code. It may take some time before the software is released to users and data starts showing up.`;
+};
```
[the remark this answers is not in the document]
Comment (wlach) [reply src/data/schemas.js:18]: These are great suggestions!

File: src/data/schemas.js [hunk 2]
```diff
@@ -107,7 +111,7 @@ export const METRIC_METADATA_SCHEMA = [
     title: "First Seen",
```
Comment (wlach) [inline src/data/schemas.js:111]:
```suggestion
    title: "First Added",
```

File: src/data/schemas.js [hunk 3]
```diff
@@ -219,7 +223,7 @@ export const PING_SCHEMA = [
...
```
Comment (wlach) [inline src/data/schemas.js:223]:
```suggestion
    title: "First Added",
```

**Comment by wlach** [comment]
Thinking about it a bit more, I think we should use "first added" instead of "first seen" as a title.

**Review by Iinh — APPROVED** [review]

**Pull request merged**